src/plugins/api-server/backend/routes/websocket.ts

```
@@ -56,6 +56,18 @@ export const register = (
     );
   };
 
+  // Notify all WebSocket clients that playback has stopped (e.g. app closing/hiding)
+  // We intentionally do NOT close or clear sockets here so that clients like
+  // Boring Notch keep their connection alive and can receive updates when
+  // Pear Desktop comes back from hiding.
+  wsNotifyClose = () => {
+    console.log(`[API Server WebSocket] NotifyClose called. Sending isPlaying:false to ${sockets.size} clients.`);
```

ESLint

[error] Replace ``[API·Server·WebSocket]·NotifyClose·called.·Sending·isPlaying:false·to·${sockets.size}·clients.`` with `⏎······`[API·Server·WebSocket]·NotifyClose·called.·Sending·isPlaying:false·to·${sockets.size}·clients.`,⏎····`

(prettier/prettier)